### ISSUE TITLE
fatal: cannot run ssh: No such file or directory

### ISSUE THREAD

**Issue by jschueller** [conda-forge/staged-recipes#7709]
Linux builds fail with current master, see latest prs
```
git fetch --force origin master:master
fatal: cannot run ssh: No such file or directory
fatal: unable to fork
Exited with code 1
```

goes on if i revert d05089e28fde15ce59d2183c6185bacdaf522a28
cc @mariusvniekerk

**Comment by mariusvniekerk** [comment]
We might need to hide that behind the azure env flag

**Comment by mariusvniekerk** [comment]
Fixed in #7711
